firestore-bigquery-export/scripts/gen-schema-view/src/__tests__/fixtures/sql/fullSchemaLatest.txt

```diff
@@ -60,6 +64,24 @@ FROM
                 timestamp DESC
             )
           ) AS last_login,
+          SAFE_CAST(
```

Comment:
These updates will have to be changed once this PR is merged in: https://github.com/firebase/extensions/pull/357